user/serializers.py

```diff
@@ -22,3 +25,42 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.EmailField(
+        label=_("Email"),
+        write_only=True
+    )
+    password = serializers.CharField(
+        label=_("Password"),
+        style={"input_type": "password"},
+        trim_whitespace=False,
+        write_only=True
+    )
+    token = serializers.CharField(
+        label=_("Token"),
+        read_only=True
+    )
+
+    def validate(
+            self,
+            attrs: Dict[str, Union[str, Any]]
```

Comment:
Union[str, Any] is so strange)
Because Any is already str
Check in other places